news/2023/2023-11-21-osu-world-cup-2023-semifinals-recap.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "osu! World Cup 2023: Semifinals Recap"
+date: 2023-11-21 18:00:00 +0000
+---
+
+As the osu! World Cup is nearing its conclusion, close matches and stupendous performances are pumping adrenaline into players and spectators alike.
```

Comment:
```suggestion
As the osu! World Cup is nearing its conclusion, close matches and stupendous performances are a constant source of adrenaline-pumping moments for both players and spectators alike! Catch a rundown of what you might've missed!
```

Reply:
i'm going for a specific tone so i will take just the first sentence